packages/react-components/src/components/chart/hooks/useTrendCursors.ts

```diff
@@ -13,12 +13,12 @@ const useTrendCursors = ({
   series,
   yMax,
   yMin,
-  chartId,
+  // chartId,
```

Comment:
can we uncomment this ? we can remove this launch if its logging too much.

Reply:
ahh yup, commented it out so I could see my own debug logs :D